client/boot/common.js

```diff
@@ -22,6 +22,9 @@ import { setLocale, setLocaleRawData } from 'state/ui/language/actions';
 import { setCurrentUserOnReduxStore } from 'lib/redux-helpers';
 import { installPerfmonPageHandlers } from 'lib/perfmon';
 import { getSections, setupRoutes } from 'sections-middleware';
+import { checkFormHandler } from 'lib/protect-form';
+import notices from 'notices';
+import authController from 'auth/controller';
```

Comment:
Hah, the `notices` and `authController` modules still use `export default = { ... }` :)

Reply:
just double checking: this comment is just an observation and doesn't imply that i should make a change here right?

Reply:
Yes, it can wait for a separate PR.